frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -200,15 +245,19 @@ export default function CourseDescription({ structure }) {
                 </Dropdown.Button>
               )}
             </div>
-            {course.is_legacy && <Text strong> NOTE: this course is discontinued - if an equivalent course exists for it, please pick that instead </Text>}
+            {course.is_legacy && (
+              <Text strong>
+                {" "}
```

Comment:
Could you remove `{" "}` if the space is not needed? Would be better style wise!